.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "eslint.validate": ["javascript", "javascriptreact", "typescript", "typescriptreact"],
+  "eslint.workingDirectories": [
+    {"directory": "packages/rps", "changeProcessCWD": true},
+    {"directory": "packages/jest-gas-reporter", "changeProcessCWD": true},
+    {"directory": "packages/wallet", "changeProcessCWD": true}
+  ],
+  "editor.formatOnSave": true
+}
```

Comment:
This file seems like it should not be in this PR.

Reply:
I think this is included because I asked for it [here](https://github.com/statechannels/monorepo/pull/573#issuecomment-559740236). I'm keen to make sure we commit any settings that are required for the setup to work, so that everything Just Works out of the box. We might be able to get away without this though, depending on whether it's truly required or not.

Reply:
There is a separate PR for this, though: https://github.com/statechannels/monorepo/pull/607

Reply:
And the one in this PR seems to be making changes related to `rps` and `jest-gas-reporter`, too.